kbase/narrative
# Dataup 187 move globus upload below drag and drop

# Description of PR purpose/changes

This PR moves the upload with globus and upload with url buttons bellow the dropzone. It also edits the text in the drop zone.
![Screen Shot 2020-10-12 at 3 05 10 PM](https://user-images.githubusercontent.com/56279459/95797634-727a9600-0c9c-11eb-96d0-816024b636e7.png)

# Jira Ticket / Issue #
https://kbase-jira.atlassian.net/browse/DATAUP-187

# Testing Instructions
* Details for how to test the PR: 
Unit tests: execute `make test-frontend-unit`

To view changes locally:
1. navigate to the narrative repo and execute `kbase-narrative`
2. create a new narrative
3. In the new narrative click the add data button and open the import tab
4. You will see the changes in on the import tab

# Dev Checklist:


# Updating Version and Release Notes (if applicable)
